test/e2e/framework/statefulset_utils.go

```diff
@@ -756,7 +756,7 @@ func NewStatefulSetPVC(name string) v1.PersistentVolumeClaim {
 			},
 			Resources: v1.ResourceRequirements{
 				Requests: v1.ResourceList{
-					v1.ResourceStorage: *resource.NewQuantity(1, resource.BinarySI),
+					v1.ResourceStorage: resource.MustParse("1Gi"),
```

Comment:
What is the reason for this change ?

Reply:
As described in the PR description, many persistent volume classes (including the default ones on DigitalOcean manage Kubernetes and EKS) don't support creating a 1-byte volume like this was previously doing. Both DO and EKS have a 1GiB minimum, so that's what I've used here. As I mentioned, we could parameterize this for more flexibility but 1GiB doesn't seem like an onerous requirement.